### ISSUE TITLE
iPad issues

### ISSUE THREAD

**Issue by eharris92** [jmoenig/Snap--Build-Your-Own-Blocks#1982]
Hello,

I'm a teacher that uses Snap in class and all my students have iPad Pros. Some students have the page randomly reload even after they have other apps closed, less tabs open, use a different browser, restart their iPad,  etc. Yet, I have one student where no matter what technique is used in an attempt to not use other memory, Snap will still reload.

I know this is very vague, but could I maybe have some insight as to why Snap keeps reloading.

Thank you

**Comment by jmoenig** [comment]
I've noticed this issue in more recent versions of iOS. Apple has made Safari increasingly slower and less stable on iOS over the past years, while refusing other browser engines.

**Comment by izzyplante** [comment]
Snap! is working quite well on newer ipads as of 2025. However I cannot record audio, even though permissions have been given for the Snap! site. I've also tested with locally hosted versions, but have not yet had a chance to look at the console log as I don't have my Safari debugging console setup on my ipad.

**Comment by jmoenig** [comment]
yes, sound issues are currently a problem on iPads and also in Safari/Mac. I'm actually working on them as we speak, but ... it's not yet clear whether I'll get them ironed out...
